### PR TITLE
Ruby 3 and Rails 7

### PR DESCRIPTION
**Pull request by elohanlon**
- HYACINTH-966
- HYACINTH-967
- HYACINTH-968

### CODE REVIEW

File: config/initializers/filter_parameter_logging.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 # Be sure to restart your server when you modify this file.
 
-# Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+# Configure parameters to be filtered from the log file. Use this to limit dissemination of
+# sensitive information. See the ActiveSupport::ParameterFilter documentation for supported
+# notations and behaviors.
+Rails.application.config.filter_parameters += [
+  :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn
```
Comment (JackBlackLight) [inline config/initializers/filter_parameter_logging.rb:7]: just doublechecking that you intended ':passw' and not ':password'

Comment (elohanlon) [reply config/initializers/filter_parameter_logging.rb:7]: I did, thanks!  That was automatically added by the `rails app:update` task.  Good idea checking though.

**Review by JackBlackLight — APPROVED** [review]
Looks good, @elohanlon. I also ran hysync:ci locally and everything passed.

**Comment by elohanlon** [comment]
Thanks, @JackBlackLight !

**Pull request merged**